public/app/features/alerting/unified/components/import-to-gma/ConfirmConvertModal.tsx

```diff
@@ -271,6 +281,35 @@ export function filterRulerRulesConfig(
   return { filteredConfig, someRulesAreSkipped };
 }
 
+/*
+This function is used to check if the rule should be skipped.
+It checks if the rule has the '__grafana_origin' label, and if the rule is from synthetics.
+If the rule has the '__grafana_origin' label, it is skipped.
+If the rule is from synthetics, it is skipped.
+*/
+function shouldSkipRule(rule: RulerRuleDTO): boolean {
+  // check if the rule has the '__grafana_origin' label
+  const hasGrafanaOriginLabel = rule.labels?.[GRAFANA_ORIGIN_LABEL];
+  if (hasGrafanaOriginLabel) {
+    return true;
+  }
+  // check if the rule is from synthetics
+  const hasSyntheticsLabels = rule.labels?.namespace === 'synthetic_monitoring';
+  if (rulerRuleType.dataSource.rule(rule)) {
+    let hasSyntheticsRuleName = false;
+    if (rulerRuleType.dataSource.alertingRule(rule)) {
+      hasSyntheticsRuleName = SYNTHETICS_RULE_NAMES.some((name) => rule.alert?.includes(name));
+    } else {
+      // recording rule
+      hasSyntheticsRuleName = SYNTHETICS_RULE_NAMES.some((name) => rule.record?.includes(name));
+    }
+    if (hasSyntheticsRuleName && hasSyntheticsLabels) {
+      return true;
+    }
+  }
```

Comment:
I think we could simplify this a bit
```suggestion
  const hasSyntheticsLabels = rule.labels?.namespace === 'synthetic_monitoring';
  
  if (!hasSyntheticsLabels) {
    return false
   }
  
  const ruleName = getRuleName(rule)
  
  return SYNTHETICS_RULE_NAMES.some((name) => name === ruleName);
  }
```